examples/c/put_var.c

```diff
@@ -0,0 +1,397 @@
+/**
+ * @file 
+ * @brief A simple C example for the ParallelIO Library using PIOc_put_vara()
+ *
+ * This example creates a netCDF output file with one dimension and
+ * one variable. It first writes, then reads the sample file using the
+ * ParallelIO library. 
+ *
+ * This example can be run in parallel for 1, 2, 4, 8, or 16
+ * processors.
```

Comment:
Can you add your name as author to any files or functions that you add? Use the @author tag.